pom.xml

```diff
@@ -77,6 +100,7 @@
         <dependency>
             <groupId>org.osgi</groupId>
             <artifactId>org.osgi.core</artifactId>
+            <version>4.2.0</version>
```

Comment:
Please use the managed artifacts from the bundle-parent

Reply:
sling parent 39 does not have a managed version of org.osgi.core

Reply:
See above

Reply:
This is managed in parent v34. Do we really want to support OSGi R4.2?

Reply:
This is managed in parent v34. Do we really want to support OSGi R4.2?

Reply:
OSGi R 4.2 is the minimum OSGi API version that this code needs. It does not require anything newer than that right now so why increase the APIs that it uses higher than what it actually needs? It will work on any OSGi version after that.